Sources/TuistKit/Services/DependenciesService.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import TSCBasic
+import TuistCore
+import TuistDependencies
+import TuistLoader
+import TuistSupport
+
+final class DependenciesService {
+    private let dependenciesController: DependenciesControlling
+    private let dependenciesModelLoader: DependenciesModelLoading
+
+    init(dependenciesController: DependenciesControlling = DependenciesController(),
+         dependenciesModelLoader: DependenciesModelLoading = DependenciesModelLoader())
+    {
+        self.dependenciesController = dependenciesController
+        self.dependenciesModelLoader = dependenciesModelLoader
+    }
+
+    func run(path: String?, method: InstallDependenciesMethod) throws {
+        logger.info("Start installing dependencies.", metadata: .section)
```

Comment:
Nitpick: `Fetching dependencies`
_(to be consistent with the command, which is named `fetch`)_